### PR TITLE
Fix phpstan findings

### PR DESCRIPTION
**Pull request by core23**
## Subject

You can test the release using this repo: https://github.com/core23/sonata-block-major

I am targeting this branch, because this is (hopefully) the last PR for before the major release.

## Changelog

```markdown
### Added
- Add support for twig 3

### Changed
- Make command fully lazy

### Fixed
- Fix some phpstan findings
```

### CODE REVIEW

File: src/Command/DebugBlocksCommand.php [hunk 1]
```diff
@@ -43,16 +40,16 @@ public function __construct(string $name = null, BlockServiceManagerInterface $b
 
     public function configure(): void
     {
-        $this->setName(static::$defaultName); // BC for symfony/console < 3.4.0
+        $this->setName(static::$defaultName);
```
Comment (OskarStark) [inline src/Command/DebugBlocksCommand.php]: Couldn’t this then be removed completely?

**Comment by franmomu** [comment]
What about removing the `$name` parameter from `DebugBlocksCommand` constructor and from [its definition](https://github.com/sonata-project/SonataBlockBundle/blob/master/src/Resources/config/commands.xml#L5).

**Comment by core23** [comment]
> What about removing the `$name` parameter from `DebugBlocksCommand` constructor and from [its definition](https://github.com/sonata-project/SonataBlockBundle/blob/master/src/Resources/config/commands.xml#L5).

Done

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**